integrationtest/showcase/library/src/main/java/org/hibernate/search/integrationtest/showcase/library/analysis/LibraryAnalysisConfigurer.java

```diff
@@ -6,25 +6,9 @@
  */
 package org.hibernate.search.integrationtest.showcase.library.analysis;
 
-import org.hibernate.search.backend.elasticsearch.analysis.ElasticsearchAnalysisConfigurer;
-import org.hibernate.search.backend.elasticsearch.analysis.model.dsl.ElasticsearchAnalysisDefinitionContainerContext;
+public abstract class LibraryAnalysisConfigurer {
```

Comment:
I was a bit surprised by the fact you define an common superclass just to define constants, but ok. 
That being said, sonar complains because this class looks like a "constants" class, which should be final and define a private constructor.

Maybe you could just rename this class to `LibraryAnalyzers`, make it final, add a private constructor, and not extend it from the configurers?